examples/gardener/main.go

```diff
@@ -13,10 +13,10 @@ import (
 )
 
 func main() {
-	projectName := flag.String("p", "", "Gardener project name")
+	projectName := flag.String("p", "garden-omar", "Gardener project name")
 	machineType := flag.String("m", "n1-standard-4", "GCP machine type")
-	credentials := flag.String("c", "", "Path to the credentials file")
-	secret := flag.String("s", "", "Name of the secret to access the underlying provider of gardener")
+	credentials := flag.String("c", "C:\\Users\\D074188\\.kube\\config", "Path to the credentials file")
+	secret := flag.String("s", "my-gcp", "Name of the secret to access the underlying provider of gardener")
```

Comment:
Please remove any personal or private data from the code.

Reply:
removed